### DISCUSSION TITLE
Git and GitHub Workflow | 2nd Issue Ticket Task | Working with GitHub web UI

### DISCUSSION THREAD

**Discussion by Ifycode** [collabo-community/.github#3] [Git and GitHub Workflow "Good First Issue" Tickets]
#### Prerequisite

This 2nd task issue ticket is based on the 1st issue ticket in this Git and GitHub workflow series. To proceed with this 2nd task, ensure that you have submitted a pull request for the 1st task, and your 1st task's pull request has been approved. 

If you have not done the 1st task, you can find it through the project board here: [git and github workflow good first issue tickets](https://github.com/orgs/collabocate-osc/projects/1?filterQuery=label%3A%22For%3A+Community+%5BGeneral%5D%22+label%3A%22topic%3A+good+first+issue%22+label%3A%22topic%3A+git+%26+github+workflow%22)

If your 1st task's pull request has been submitted and approved by a maintainer, you can proceed to do this 2nd task.

#

#### Helpful resource
This youTube playlist will give you an idea of how to work with GitHub web User Interface: https://youtube.com/playlist?list=PLMDhbo3xlD1ESa1_9WN4yG7bGDDksEQx7&si=F9lzacG1cr9WGS-M

#

#### Issue description

In this **2nd task assignment**, you are required to make some changes to the markdown tutorial file that you created in the 1st task **from the GitHub web User Interface**. It will also involve learning **how to merge conflicts**. Find more information in the expected solution section below.

#

#### Expected solution

> [!TIP]  
> You are to sync your fork, create your branch, and make the changes to the already existing files or folders using the GitHub web User Interface.

- You may have to first sync your fork's `develop` branch with the @collabocate-osc repo's `develop` branch, so that your 1st task changes that we merged into our develop branch can reflect in your fork's develop branch.
- Create a new branch from the `develop` branch. Your branch name should be `@GH-replaceThisPartAfterTheHyphenWithTheIssueNumberForThisTicket` (don't add the # sign to the ticket number, and no space in there). It is in this new branch that you should make your changes in and submit from.
- In the`gitcollab-ui-task.md` file that you previously created (in the 1st task), change this line:
````
- Inline Code Block: `This code block text will show up inline`.
````
To this instead:
````
- Inline Code Block (with non-code text): `This code block text will show up inline.` This part of the text is just ordinary text!
````
- Once you are done, request for review as usual. We will guide you to know when a conflict has occurred in your pull request, and steps to fixing it.

#

#### Additional context
- Let us know in the comment of this issue ticket (GitHub discussion) that you will be participating in this 2nd task assignment. Ensure to tag @collabocate-osc/maintainers in your comment. That way, maintainers will be notified and will be prepared to help review your pull request when you submit.
- Repo URL (where you are to do and submit this task to): https://github.com/collabocate-osc/good-first-issue-GIT-solutions
- If you have any questions or need any clarification when working on this ticket, please use the `#git-workflow-support` channel in our [discord community server](https://resources.collabocate.community).
- To help you better understand how to work with other contributors within @collabocate-osc better, we will review your pull request based on the directions given in this issue ticket and our [Public Documentation Resource Guide](https://resources.collabocate.community).

#

#### Testing checklist

Note (to contributor taking on this task assignment): Add the testing checklist below (without this note) to your pull request when submitting your fix. This will guide (you and) the contributors who will review your fix to know what is important to check or test for.
- [ ] Here's a link to my approved 1st task pull request: `Replace this inline code block with the link to your approved 1st task pull request`
- [ ] My branch for this task assignment is created from the `develop` branch, and the name of my branch is a mix of `@GH-` and the ticket's issue number (no # sign and no space in there).
- [ ] I have updated the `gitcollab-ui-task.md` file in my own folder, as directed in the 2nd issue ticket task.
- [ ] I certify that I ran my checklist.

**Comment by Goodnews-code** [comment]
Hi @collabo-community/maintainers, i will be participating in this 2nd task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by Yanmi01** [comment]
Hi @collabo-community/maintainers

I will be participating in this second issue ticket task

**Reply by Ifycode** [reply]
👍🏼

**Comment by BA-atomic** [comment]
Hi @collabo-community/maintainers, i will be participating in this 2nd task assignment.

**Reply by Akpjunior94** [reply]
:+1:

**Comment by OpeyemiAdeniji** [comment]
Hi @collabo-community/maintainers, i will be participating in this 2nd task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by Oyebliss** [comment]
Hi @collabo-community/maintainers, 
I will be participating in this 2nd task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by KBandipo** [comment]
Hi @collabo-community/maintainers 
I will be participating in this 2nd task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by Danbaba1** [comment]
Hi @collabo-community/maintainers
I will be participating in this 2nd task assignment.

**Reply by samuko-things** [reply]
ok @Danbaba1

**Comment by elizabethobiagba** [comment]
Hello @collabo-community/maintainers I will be participating in this 2nd task assignment

**Reply by Ifycode** [reply]
👍🏼

**Comment by oluwadamilola21** [comment]
Hi @collabo-community/maintainers
I will be participating in this 2nd task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by Ifycode** [comment]
> [!IMPORTANT]  
> The name of our community has changed from `collabo-community` to `collabocate-osc`. Please tag maintainers with @collabocate-osc/maintainers from now on. Thank you.

**Comment by Yanmi01** [comment]
Hi @@collabocate-osc/maintainers, I will be participating in this 2nd task assignment

**Reply by Ifycode** [reply]
👍🏼
